frontend/packages/dev-console/integration-tests/features/addFlow/add-page.feature

```diff
@@ -0,0 +1,95 @@
+              And user will see "From Local Machine" card
+              And user will see "Serverless" card
+        
+        
+        @regression
+        Scenario: Developer Catalog option to create an Application, Component and Service
+             Then user will see "All Services" option
+              And user will see "Database" option
+              And user will see "Operator Backed" option
+              And user will see "Helm Chart" option
+              And user will see "Event Source" option
+
+        
+        @regression
+        Scenario: Git Repository option to create an Application, Component and Service
+             Then user will see "From Git" option
+              And user will see "From Devfile" option
+              And user will see "From Dockerfile" option
+
+        
+        @regression
+        Scenario: From Local Machine option to create an Application, Component and Service
+             Then user will see "Import YAML" option
+              And user will see "Upload JAR file" option
+
+        
+        @regression
+        Scenario: Details Toggle on
+             When user enable Details toggle
+             Then user will see label Details on
+              And user will see description "Browse the catalog to discover, deploy and connect to services"
+
+
+        @regression
+        Scenario: Details Toggle off
+             When user disable Details toggle
+             Then user will see label Details off
+              And user will not see description "Browse the catalog to discover, deploy and connect to services"
+
+
+        @regression
+        Scenario: Hide Getting Started Resources from View
+            Given getting started resources is not hidden from view
+             When user clicks on kebab menu
+              And user selects Hide from view option
+             Then user will not see Getting started resources card
+
+        
+        @regression
+        Scenario: Show Getting Started Resources
+            Given getting started resources is hidden from view
+             When user clicks on Show getting started resources link
+             Then user will see Getting started resources card
+          
+     
+        @regression
+        Scenario: Close Show Getting Started Resources link
+            Given getting started resources is hidden from view
+             When user clicks on close Show getting started resources link
+             Then user will not see Show getting started resources link
+
+          
+        @regression @manual
+        Scenario: Getting Started Resources card when all Quick Starts have been completed
+            Given user has completed all Quick Starts
+             Then user will not see Build with guided documentation at Developer perspective on getting started resources card
+              And user will not see Build with guided documentation at Admiinistrator perspective on getting started resources card
```

Comment:
typo:
`Admiinistrator` -> `Administrator`

Reply:
Also, should we include the case for the admin perspective in the tests for developer perspective?

Reply:
This is actually out of scope. But this is a manual scenario, that's why I have added that statement.